pkg/scheduler/factory/factory.go

```diff
@@ -634,6 +634,9 @@ func (c *configFactory) updatePodInCache(oldObj, newObj interface{}) {
 		return
 	}
 
+	// NOTE: Because the scheduler uses snapshots of schedulerCache and the live
+	// version of equivalencePodCache, updates must be written to schedulerCache
+	// before equivalencePodCache.
```

Comment:
I would change this to "before invalidating equivalencePodCache".

Reply:
Done